client/dashboard/sites/scan/index.tsx

```diff
@@ -31,6 +33,22 @@ function SiteScan( { scanTab }: { scanTab: 'active' | 'history' } ) {
 	const router = useRouter();
 
 	const { data: site } = useSuspenseQuery( siteBySlugQuery( siteSlug ) );
+	const { data: scan } = useQuery( siteScanQuery( site.ID ) );
```

Comment:
Nit: Using `useQuery` might cause the message "Last automated scan ran..." render afterwards, thus causing a layout shift.

Reply:
I think we could just use `useSuspenseQuery` for this one as well.